docs/content_management/locations.md

```diff
@@ -89,33 +89,33 @@ Location visibility allows you to control which parts of the content tree are av
 
 ![Location visibility](content_management_visibility.png "Location visibility")
 
-Once a Content item is published, it cannot be un-published.
-When the Location of a Content item is hidden, the system will not display it on the website.
+Once a content item is published, it cannot be un-published.
+When the Location of a content item is hidden, the system will not display it on the website.
 
 !!! caution "Visibility and permissions"
 
     The [visibility switcher](https://doc.ibexa.co/en/latest/content_management/locations/#location-visibility) is a convenient feature for withdrawing content from the frontend.
     It acts as a filter in the frontend by default. You can choose to respect it or ignore it in your code.
     It isn't permission-based, and **doesn't restrict access to content**. Hidden content can be read through other means, like the REST API.
 
-    If you need to restrict access to a given Content item, you could create a role that grants read access for a given [**Section**](sections.md) or
+    If you need to restrict access to a given content item, you could create a role that grants read access for a given [**Section**](sections.md) or
     [**Object State**](object_states.md), and set a different Section or Object State for the given Content.
     Or use other permission-based [**Limitations**](limitations.md).
 
-If a Content item is hidden, it is invisible in all its Locations.
+If a content item is hidden, it is invisible in all its Locations.
 If a Location is hidden, all of its descendants in the tree will be hidden as well.
 This means that there are three different visibility statuses:
 
 - Visible
 - Hidden
 - Hidden by superior
 
-All Locations and Content items are visible by default.
+All Locations and content items are visible by default.
 If a Location is made invisible manually, its status is set to Hidden.
 All Locations under it will change status to Hidden by superior.
-A Content item is Hidden by superior only in Locations in which it has a parent Location with the Hidden status.
+A content item is Hidden by superior only in Locations in which it has a parent Location with the Hidden status.
 
-In the following example, the **Content item 1** is Hidden by superior in the **Location A** while still visible in the **Location B**.
+In the following example, the **content item 1** is Hidden by superior in the **Location A** while still visible in the **Location B**.
```

Comment:
I wouldn't downcase this one as this is the label in the schema/illustration [locations_visibility.png](https://github.com/ibexa/documentation-developer/blob/IBX-7909/docs/content_management/img/locations_visibility.png)
```suggestion
In the following example, the **Content item 1** is Hidden by superior in the **Location A** while still visible in the **Location B**.
```